<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="XSHD--20161124D" name="appium.AppiumStartAndStop" tests="2" failures="0" timestamp="23 五月 2018 15:24:32 GMT" time="0.017" errors="1">
  <testcase name="setUp" time="0.017" classname="appium.AppiumStartAndStop">
    <error type="java.io.IOException" message="Cannot run program &quot;C:/Users/Administrator/AppData/Roaming/npm/node_modules/appium/build/lib/appium.js&quot;: CreateProcess error=193, %1 不是有效的 Win32 应用程序。">
      <![CDATA[java.io.IOException: Cannot run program "C:/Users/Administrator/AppData/Roaming/npm/node_modules/appium/build/lib/appium.js": CreateProcess error=193, %1 不是有效的 Win32 应用程序。
	at java.lang.ProcessBuilder.start(Unknown Source)
	at java.lang.Runtime.exec(Unknown Source)
	at java.lang.Runtime.exec(Unknown Source)
	at java.lang.Runtime.exec(Unknown Source)
	at appium.AppiumStartAndStop.appiumStart(AppiumStartAndStop.java:30)
	at appium.AppiumStartAndStop.setUp(AppiumStartAndStop.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.TestRunner.beforeRun(TestRunner.java:626)
	at org.testng.TestRunner.run(TestRunner.java:594)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.io.IOException: CreateProcess error=193, %1 不是有效的 Win32 应用程序。
	at java.lang.ProcessImpl.create(Native Method)
	at java.lang.ProcessImpl.<init>(Unknown Source)
	at java.lang.ProcessImpl.start(Unknown Source)
	... 29 more
]]>
    </error>
  </testcase> <!-- setUp -->
  <testcase name="ScrollToView" time="0.000" classname="appium.AppiumStartAndStop">
    <skipped/>
  </testcase> <!-- ScrollToView -->
</testsuite> <!-- appium.AppiumStartAndStop -->
